dbms/tests/queries/0_stateless/01035_avg_weighted.sql

```diff
@@ -0,0 +1 @@
+SELECT avgWeighted(x, weight) FROM (SELECT t.1 AS x, t.2 AS weight FROM (SELECT arrayJoin([(1, 1), (2, 1), (3, 1), (4, 1), (5, 1)]) AS t));
```

Comment:
Well, this test is not interesting if all weights are 1. Need test with different weights. And with all weights = 0.

Reply:
And also need test with different types of x and weight (which could pass or throw specified exception).